BRANDING.md

```diff
@@ -0,0 +1,55 @@
+
+## Purpose of the Brand Guidelines
+    
+These guidelines exist to ensure that the Istio project can share its platform under open source licenses while making sure that the “Istio” brand is protected as a meaningful source identifier in a way that is consistent with trademark law.
+ 
+## Acceptable Uses
+
+Given the open nature of Istio, you may use the Istio trademark to refer to the project without prior written permission. Examples of these acceptable references include the following:
+
+-   To refer to the Istio project itself;
+    
+-   To refer to unmodified source code or other files shared by the Istio repositories on GitHub;
+    
+-   To accurately identify that your design or implementation is based on or is for use with the Istio platform.
+    
+-   To discuss the Istio project for educational or informational purposes
+    
+*Examples:*
+
+-   “[Your Product/Offering] for Istio.”
+    
+-   “[XYZ] Conference for Istio Users.”
+    
+-   “Making [XYZ] better with Istio”
+    
+## Usage Guidelines
+
+-   The Istio name may not be used or registered in a manner that would cause confusion as to the Istio project’s affiliation or endorsement.
+    
+-   Don’t use the Istio name as part of your company name, product name, domain name, or social media profile.
+    
+-   Other than as permitted by these guidelines, the Istio name should not be combined with other trademarks, terms, or source identifiers.
+    
+-   Don't remove, distort or alter any element of the Istio Trademarks without permission:
+    
+-   Do not shorten, abbreviate, or change the spelling of the Istio name, for example in mass communication, product names, social media profiles, domain names or event names).
+    
+-   Do not change the color or design elements of the Istio logo, or combine the logo with any other text.
+    
+-   Community websites, social media accounts, or other community-organized properties or organizations should follow the naming convention "[XYZ] for Istio Users" or "[XYZ] for Istio Contributors".
+    
+-   The name "Istio Working Group" can only be used by approved working groups maintaining code in the Istio GitHub organization under Istio Project governance.
+    
+-   If you’re organizing a conference, training, or certification program, you can use the word Istio to refer to the technology that the event or program covers. However, Istio cannot be contained in the title name for your conference or program (such as IstioConf or Certified Istio Administrator) without further explanation. An example of an appropriate usage could be “XYZ Conference for Istio Users” or “XYZ Certification Program for Istio Users” with XYZ being your own brand name. Any related materials should also prominently disclaim any official affiliation with or endorsement by the Istio project.
+    
+-   Don’t use the Istio logo on any website, product UI, or promotional materials without our prior written permission, except as specified above.
```

Comment:
If anyone wanted to put the Istio logo on their website, and it's not one of the examples covered earlier in the Acceptable Uses area - then they would just need to email Steering with the request. So if someone wanted to use the logo to showcase their product was built on Istio, that would be okay.

Reply:
The logo usage is not clear to me still. "as specified above" doesn't really point to much in regard to the logo. For example. For example, lets say I have a product called  "XYZ Service Mesh with Istio". That followed the naming guidance. Does that imply I can use the Istio logo on the product page?   If not we may need some more explicit language on where the logo can be used without written permission.

Reply:
It might clear it up if we have one section for the name and one for the logo, then each section can have a list of "acceptable" uses/examples and a list of "unacceptable" usages/examples.  Then this bullet can go into the "logo" section and "as specified above" would be "except as specified in the 'acceptable' section".

Reply:
Possible wording change: "Do not use the Istio mark on posters, brochures, signs, websites, or other marketing materials to promote your events, products or services without written permission from Istio Steering"

Reply:
This comment thread informed the new language. Closing this as resolved.